Makefile

```
@@ -60,33 +60,43 @@ SPECTEST_VERSION := $(shell cat .spectest_version)
 SPECTEST_CONFIGS = general minimal mainnet
 
 SPECTEST_ROOTDIR = test/spec/vectors
+SPECTEST_GENERATED_ROOTDIR = test/generated
+VECTORS_DIR = $(SPECTEST_ROOTDIR)/tests
 # create directory if it doesn't exist
 $(info $(shell mkdir -p $(SPECTEST_ROOTDIR)))
 
 SPECTEST_DIRS := $(patsubst %,$(SPECTEST_ROOTDIR)/tests/%,$(SPECTEST_CONFIGS))
+SPECTEST_GENERATED := $(patsubst %,$(SPECTEST_GENERATED_ROOTDIR)/%,$(SPECTEST_CONFIGS))
 SPECTEST_TARS := $(patsubst %,$(SPECTEST_ROOTDIR)/%_${SPECTEST_VERSION}.tar.gz,$(SPECTEST_CONFIGS))
 
 $(SPECTEST_ROOTDIR)/%_${SPECTEST_VERSION}.tar.gz:
 	curl -L -o "$@" \
 		"https://github.com/ethereum/consensus-spec-tests/releases/download/${SPECTEST_VERSION}/$*.tar.gz"
 
-$(SPECTEST_ROOTDIR)/tests/%: $(SPECTEST_ROOTDIR)/%_${SPECTEST_VERSION}.tar.gz
+$(VECTORS_DIR)/%: $(SPECTEST_ROOTDIR)/%_${SPECTEST_VERSION}.tar.gz
 	-rm -rf $@
 	tar -xzmf "$<" -C $(SPECTEST_ROOTDIR)
 
+$(SPECTEST_GENERATED_ROOTDIR): $(VECTORS_DIR)
```

This results in "there's no target to generate `test/spec/vectors/tests`". We could make this depend on all subdirs:
```suggestion
$(SPECTEST_GENERATED_ROOTDIR)/%: $(VECTORS_DIR)/%
```
And then use `$(SPECTEST_GENERATED)` instead of `$(SPECTEST_GENERATED_ROOTDIR)` in the remaining targets.

---

Changed this so it depends on each vector dir and on the script.